text/0000-target-tier-policy.md

```diff
@@ -0,0 +1,685 @@
+for demotion of a tier 1 target (with or without host tools) requires a full
+RFC process, with approval by the compiler and release teams. Any such proposal
+will be communicated widely to the Rust community, both when initially proposed
+and before being dropped from a stable release.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+The set of approving teams for each tier arose out of discussion with the
+various teams involved with aspects of the Rust project impacted by new
+targets.
+
+Policies that require the approval of multiple teams could instead require a
+core team approval. This would have the advantage of reducing the number of
+people involved in the final approval, but would put more coordination effort
+on the core team and the various team leads to ensure that the individual teams
+approve. As another alternative, we could separate the individual team
+approvals (into separate issues or separate rfcbot polls), to simplify checking
+for consensus and reduce diffusion of responsibility; however, this could also
+increase the resulting complexity and result in discussions in multiple places.
+
+We could introduce specific time requirements for the amount of time a target
+must spend at a tier before becoming eligible for promotion to a higher tier.
+
+We could renumber tiers, rather than having "tier 2 with host tools" and "tier
+1 with host tools". However, some targets may by design never reach "tier 1
+with host tools", and this would make such targets seem deficient or
+second-class compared to whatever "tier 1" ended up being called. In addition,
+people already widely use the existing tier 1/2/3 numbering, and anything
+changing that numbering would introduce confusion; such a renumbering seems
+unlikely to provide value commensurate with that confusion.
+
+# Prior art
+[prior-art]: #prior-art
+
+This attempts to formalize and document Rust policy around targets and
+architectures. Some requirements of such a policy appear on the [Rust Platform
+Support page](https://forge.rust-lang.org/release/platform-support.html).
+
+Future expansions of such policy may find requirements from other communities
+useful as examples, such as Debian's [arch
+policy](https://release.debian.org/bullseye/arch_policy.html) and [archive
+criteria](https://salsa.debian.org/ftp-team/website/-/blob/HEAD/archive-criteria.html).
+
+Other precedents for tiered target support include [Firefox's supported build
+targets](https://firefox-source-docs.mozilla.org/build/buildsystem/supported-configurations.html),
+[node.js supported
+platforms](https://github.com/nodejs/node/blob/HEAD/BUILDING.md#supported-platforms),
+and [GHC's platform
+support](https://gitlab.haskell.org/ghc/ghc/-/wikis/platforms).
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+- How should we track the maintainers of a target, so that we can page them if
+  we need an issue addressed involving that target?
```

Comment:
We now have an ARM and a RISC-V notification group that can be pinged via `@rustbot ping arm` / `@rustbot ping risc-v`

Reply:
I've moved this from "unresolved questions" to "prior art" and documented this precedent (without making any of the specifics into policy requirements).